addons/hr_skills_survey/views/hr_templates.xml

```diff
@@ -20,6 +20,12 @@
                     {'default_certification': True}
                 </attribute>
             </field>
+            <field name="date_start" position="attributes">
+                <attribute name="invisible">display_type == 'certification'</attribute>
+            </field>
+            <field name="date_start" position="after">
+                <field name="date_start" invisible="display_type != 'certification'" string="Completed On"/>
+            </field>
```

Comment:
You hide date_start and then add another date_start after date_start? ---> 🤯 

This is just to change the label, right?
Can't you just split the label in a separate "label" tag in the base form and then modify the label here?